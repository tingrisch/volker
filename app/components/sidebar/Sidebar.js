import React from 'react';
import styled from 'styled-components';

import Navigation from '../navigation/Navigation';

const Wrapper = styled.div`
  box-shadow: 3px 3px 4px 0px rgba(0, 0, 0, 0.45);
  background: linear-gradient(
    to bottom,
    rgba(255, 218, 186, 0.8) 0%,
    rgba(255, 159, 76, 0.9) 65%,
    rgba(255, 218, 186, 0.95) 100%
  );
  width: 178px;
  height: 600px;
`;

const Sidebar = () => {
  return <Navigation />;
};

export default Sidebar;
