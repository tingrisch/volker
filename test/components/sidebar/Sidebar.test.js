import React from 'react';
import { shallow } from 'enzyme';

import Sidebar from '../../../app/components/sidebar/Sidebar';

describe('<Sidebar />', () => {
  test('renders', () => {
    const SidebarComponent = shallow(<Sidebar />);
    expect(SidebarComponent).toMatchSnapshot();
  });
});
