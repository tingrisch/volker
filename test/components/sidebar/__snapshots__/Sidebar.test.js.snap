// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> renders 1`] = `
<styled.div>
  <Navigation />
</styled.div>
`;
